File: features/snippets/external_storage/custom_driver/custom_driver_data_converter.py
import dataclasses

from custom_storage_driver import (  # type: ignore[import-not-found]
    LocalDiskStorageDriver,
)
from temporalio.converter import DataConverter, ExternalStorage


def configure():
    # @@@SNIPSTART python-custom-driver-data-converter
    data_converter = dataclasses.replace(
        DataConverter.default,
        external_storage=ExternalStorage(
            drivers=[LocalDiskStorageDriver()],
        ),
    )
    # @@@SNIPEND
    return data_converter

File: features/snippets/external_storage/custom_driver/custom_driver_data_converter.ts
import { ExternalStorage } from '@temporalio/common';
import { LocalDiskStorageDriver } from './custom_storage_driver';
/* eslint-disable @typescript-eslint/no-unused-vars */

{
  // @@@SNIPSTART typescript-custom-driver-data-converter
  const dataConverter = {
    externalStorage: new ExternalStorage({
      drivers: [new LocalDiskStorageDriver()],
    }),
  };
  // @@@SNIPEND
}
